### DISCUSSION TITLE
Linux app crash since Air 50.2.3.1

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#2719] [Q&A] [answered]
Hello,

see this [discussion](https://forum.starling-framework.org/d/23144-how-to-build-air-game-for-linux/42) on Starling forum.
this Linux [app](https://airapport.itch.io/farm-and-mine) crash using Air 50.2.3.1 using adt and adl but work using Air 50.2.2.6 ?

Thanks

**Comment by ajwfrost** [comment, marked as answer]
Thanks -> this is being discussed under #2712  - trying to work out whether a change in the build machine was the culprit, or a change in the code!
